dev/io.openliberty.org.jboss.resteasy.mprestclient/src/org/jboss/resteasy/microprofile/client/ot/LibertyRestClientBuilderImpl.java

```diff
@@ -130,8 +134,28 @@ public class LibertyRestClientBuilderImpl implements RestClientBuilder {
     public static final ClientHeadersRequestFilter HEADERS_REQUEST_FILTER = new ClientHeadersRequestFilter();
 
     private static final Class<?> FT_ANNO_CLASS = getFTAnnotationClass();
+    private static final LibertyProxyClassLoader myClassLoader; // Liberty change
```

Comment:
Not sure if this should be static.  We are going to want different ProxyClassLoaders for different applications.

Reply:
Will be interesting to see what the impact to performance is after this change in relationship to MP rest client.  FYI @jdmcclur.